Configuration/TypoScript/ContentElement/BaseImage.typoscript

```diff
@@ -1,12 +0,0 @@
-lib.baseImage {
```

Comment:
Please do not touch this file. It is not used because of core regressions in parsing typoscript.